File: src/fibonacci/cpp/fibonacci.cpp
#include <iostream>
using namespace std;
  
void fib(long n)
{
    long a = 0, b = 1;
    while (a < n) 
    {
        cout << a << ' ';
        swap(a,b);
        b = a+b;
    }
    cout << endl;
}

int main(int argc, char *argv[])
{
    fib(stol(argv[1]));
    return 0;
}

File: src/fibonacci/js/fibonacci.js
function fib(n) {
    let a = 0, b = 1;
    while (a < n) {
        process.stdout.write(String(a)+" ");
        [ a, b ] = [ b, a+b ];
    };
    process.stdout.write("\n");
}

fib(Number(process.argv[2]));